src/lib.rs

```diff
@@ -8,10 +8,9 @@
 // - Isis Agora Lovecruft <isis@patternsinthevoid.net>
 // - Henry de Valence <hdevalence@hdevalence.ca>
 
-#![cfg_attr(not(feature = "std"), no_std)]
-
-#![cfg_attr(feature = "alloc", feature(alloc))]
+#![no_std]
```

Comment:
This provides consistent semantics between `no_std` and `std` and makes the `std` feature purely additive